src/client/app.coffee

```diff
@@ -133,10 +172,12 @@ class Player
 
 class GameView
     constructor: ->
+        @movesPerTurn = 4
```

Comment:
We should probably have a config file or something so we can tweak game mechanics in the future